trillian/ctfe/ct_server/main.go

```diff
@@ -140,6 +142,23 @@ func main() {
 		http.Handle("/metrics", promhttp.Handler())
 	}
 
+	if *getSTHInterval > 0 {
+		// Regularly update the internal STH for each log so our metrics stay up-to-date with any tree head
+		// changes that are not triggered by us.
+		for _, c := range cfg {
```

Comment:
In Go-land, goroutines are supposed to be cheap so I think one per-log makes sense.